### DISCUSSION TITLE
SDKMAN wants to join Commonhaus!

### DISCUSSION THREAD

**Discussion by marc0der** [commonhaus/foundation#136] [Joining Commonhaus]
### Project information

- **Project name**: SDKMAN!
- **Project website**: https://sdkman.io
- **Code repository**: https://github.com/sdkman/sdkman-cli
- **License**: Apache 2.0

### Describe your project

We are an OSS project that serves to empower developers. We aim to remove the mundane task of downloading and installing SDKs so developers can focus on what matters: writing code and bringing value.

SDKMAN is written in bash, but we are in the process of rewriting it in Rust. It allows developers to install SDKs with minimal fuss and effort and allows them to seamlessly switch between SDK versions. We even enable using different versions of the same SDK alongside each other across multiple terminals. The focus is on excellent user experience and making mundane tasks effortless.

The software we provide and the undergirding services are free to use. Our software is distributed under the Apache 2.0 license and may be copied and modified at will. The software comes with no warranty or guarantees.

**Comment by gavinking** [comment]
Awesome! Lots of sdkman users around here, including me.

Let's make this happen!

**Reply by marc0der** [reply]
Great stuff! What are the next steps, and what can I do to make it happen?

I also have some questions, but I'm not sure if this is the right forum to ask them.

**Reply by gavinking** [reply]
Yes this is absolutely the right place. I know @ebullient has been offline (due to conference travel) so I guess just wait until she reappears.

**Comment by cowtowncoder** [comment]
Agreed; I am a happy user as well and would really like to see sdkman join!

**Comment by alsaghir** [comment]
Exciting, here is another user for best SDK manager for macOS !

**Comment by ebullient** [comment]
@marc0der .. sorry for being so quiet. We have a lot of logistics to set up. 

Please use this discussion (not necessarily this sub-thread) to ask any other questions you may have.

- [ ] **Review the bylaws**, especially sections about [projects](https://www.commonhaus.org/bylaws/cf-membership.html#projects), [decision making](https://www.commonhaus.org/bylaws/decision-making.html), and [voting for significant decisions](https://www.commonhaus.org/bylaws/cf-council.html#voting). You may have seen some PRs go through as I work with the current EGC to create a record of approval with the newly established EGC) to ratify them and you're welcome to weigh in on these.
- [ ] **Read the [bootstrapping agreement](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md)**.
- [ ] If all is well, and you agree to participate in the bootstrapping phase to help us build this thing, then **create a PR** with the following:

    1. A filled out copy of the bootstrapping agreement. You can make a copy of [bootstrapping.md](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md) file and check that in, or checkin a filled out [pdf](https://github.com/commonhaus/foundation/releases/download/final-draft/bootstrapping.pdf). Please follow naming conventions.
    2. An updated [CONTACTS.yaml](https://github.com/commonhaus/foundation/blob/main/CONTACTS.yaml) that names your project representative in the EGC section
    3. An updated [PROJECTS.yaml](https://github.com/commonhaus/foundation/blob/main/PROJECTS.yaml) that provides details / describes your project
    4. An updated [TRADEMARKS.md](https://github.com/commonhaus/foundation/blob/main/TRADEMARKS.md) that adds your project name and project logo (if you have one). Please include the footnote reference, as we are using your project name and logo with your permission until we have a formal agreement in place and resources are transferred.

    The commit should be "Signed-off-by" your GH login (`git commit -s -m "<project> bootstrapping agreement, yyyy-mm-dd"`)

**Comment by ebullient** [comment]
Resolved by #160 

Welcome to Commonhaus! 🎉 💖 🚀
